src/music-quiz.ts

```diff
@@ -206,6 +214,8 @@ export class MusicQuiz {
                     position = ':second_place:'
                 } else if (index === 3) {
                     position = ':third_place:'
+                } else if (index > 3) {
```

Comment:
What's the issue with the scoreboard - can you show me an image of what it prints? 

It should show positions above 3 with bold: `let position = `**${index + 1}.** ``

Reply:
It shows this https://ibb.co/1nf6x62
For the other suggestion, it's possible that the code doesn't work. I'm not familiar with the language but it's an idea that can be add.